packages/clients/README.md

```diff
@@ -265,10 +279,10 @@ To use gas station with a local signer:
 
 ```ts
 import { Aptos, AptosConfig, Network } from "@aptos-labs/ts-sdk";
-import { GasStationClient } from "@shinami/clients/aptos";
+import { createAptosClient, GasStationClient } from "@shinami/clients/aptos";
 
-// Create your Aptos client targeting the desired network.
-const aptos = new Aptos(new AptosConfig({ network: Network.TESTNET }));
+// Obtain APTOS_NODE_ACCESS_KEY from your Shinami web portal.
+const aptos = createAptosClient(APTOS_NODE_ACCESS_KEY);
```

Comment:
I've decided to remove the `APTOS_` prefix because most of the time users would only be using one network so I don't think it's as necessary to preface it with the chain.